Gemfile.lock

```diff
@@ -18,7 +18,7 @@ GEM
     pry (0.12.2)
       coderay (~> 1.1.0)
       method_source (~> 0.9.0)
-    rack (2.2.3)
+    rack (3.0.8)
```

Comment:
Why on earth does this gem depend on `rack`?

Reply:
How on earth do I know? 🤷🏻‍♂️